src/renderer/services/midgard/pools.ts

```diff
@@ -62,6 +63,15 @@ const createPoolsService = (
 ): PoolsService => {
   const midgardDefaultApi$ = FP.pipe(byzantine$, liveData.map(getMidgardDefaultApi), RxOp.shareReplay(1))
 
+  const { get$: poolsFilters$, set: internalSetPoolsFilter, get: internalGetPoolsFilter } = observableState<
```

Comment:
[nitpick] `internalSetPoolsFilter` -> `_setPoolsFilter` Just to keep same naming convention for "private" things.